src/cdk-experimental/combobox/combobox.ts

```diff
@@ -278,13 +274,24 @@ export class CdkCombobox<T = unknown> implements OnDestroy, AfterContentInit {
     ];
   }
 
+  private _getPanelInjector() {
+    this._panelInjector =
+      this._panelInjector || Injector.create({providers: [], parent: this._injector});
```

Comment:
I think that you can drop the `Injector.create` here and pass the `_injector` directly.

Reply:
Hmm, I'm not sure if that's how it works. My understanding is that `@Self` skips nodes within the tree, rather than injectors. Either way, I don't think that it's blocker.

Reply:
Ah, you're probably right I'm not an expert on it. I'll just pass the injector along directly for now. In the future if we need to put more info in the injector we can create a new one